config/software/perl.rb

```diff
@@ -35,8 +35,13 @@
   env = with_standard_compiler_flags(with_embedded_path)
 
   solaris_mapfile_path = File.expand_path(Omnibus::Config.solaris_linker_mapfile, Omnibus::Config.project_root)
-  if solaris2? && File.exist?(solaris_mapfile_path)
-    cc_command = "-Dcc='gcc -static-libgcc -Wl,-M #{solaris_mapfile_path}"
+  if solaris_10?
+    cc_command = "-Dcc='gcc -static-libgcc'"
+    if File.exist?(solaris_mapfile_path)
```

Comment:
do we have to be this cautious?  i would bet money we're the only ones in the world compiling perl for solaris.

actually why are we even compiling perl for solaris?  we don't ship the server stuff with solaris...  only the client.  client doesn't use perl.

if we don't use it, per our policy that this repo is for our software, we should drop it.